android/build.gradle

```diff
@@ -288,3 +288,27 @@ tasks.register("copyCodegenArtifacts") {
 if (isGHExampleApp() && isNewArchitectureEnabled()) {
     tasks.generateCodegenArtifactsFromSchema.finalizedBy('copyCodegenArtifacts')
 }
+
+tasks.register("checkIntegrityBetweenArchitectures") {
+    group 'Verification tasks'
+    description 'Task to check integrity between fabric and paper architecture in terms of codegen generated interfaces/delegates'
+
+    if (isGHExampleApp()) {
+        return
+    }
+
+    def absoluteCodegenArtifactsPaperDestination = "../${project.property('codegenArtifactsPaperDestination')}"
+    def absoluteCodegenArtifactsSource = "../${project.property('codegenArtifactsSource')}"
```

Comment:
You would need to run the codegen task manually or build the `FabricExample` app on Android before comparing files. Otherwise, I don't think the generated files would be there.